kod/object/item/passitem/chalice.kod

```diff
@@ -29,6 +29,14 @@ resources:
    chalice_empty = "You sip the last bit of water from the Chalice of Rain."
    chalice_combine = "You pour the water from one Chalice into the other."
    chalice_cant_use_pkill = "Only innocents may partake of the chalice."
+   chalice_refilled = "The Chalice of Rain glows faintly for a moment, renewed by the spirit of Shal'ille that seems to permeate the area."
+
+   chalice_condition_exc = " filled to the brim with pure, sparkling water."
+   chalice_condition_exc_mended = " filled to the brim with pure, sparkling water."
+   chalice_condition_good = " is still plenty full."
```

Comment:
The difference between "plenty full" and "a few sips left" seems too much. Given that there are a handful of sips before it's empty, maybe it shouldn't start out "filled to the brim".

Some ideas to take or leave:
1. holds pure, potable sparkling water (too wordy but suggests consumable)
2. holds several more sips
3. has a few sips left
4. is almost empty